components/mosart/bld/namelist_files/namelist_defaults_mosart.xml

```diff
@@ -43,7 +43,7 @@ for the CLM data in the CESM distribution
 
 <frivinp_rtm rof_grid="r0125" >rof/mosart/MOSART_global_8th_20180211c.nc</frivinp_rtm>
 <frivinp_rtm rof_grid="r025" >rof/mosart/MOSART_global_qd_20240212.v3.nc</frivinp_rtm>
-<frivinp_rtm rof_grid="r05" >rof/mosart/MOSART_global_half_20180721a.nc</frivinp_rtm>
+<frivinp_rtm rof_grid="r05" >rof/mosart/MOSART_Global_half_20210422.nc</frivinp_rtm>
```

Comment:
This seems like it would change answers for v2.1 watercycle cases because they use r05 river.

Reply:
@rljacob Yes, you are correct. I found that while running tests as well. I will be reverting this change as we only want BFB changes in this PR.